portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,15 +5,36 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello there, Im Edson!</h1>");
+    ArrayList<String> thoughs = new ArrayList<String>();
```

Comment:
I believe you can get rid of the second `String` on this line, making it: 

```
ArrayList<String> thoughs = new ArrayList<>();
```

More info: https://stackoverflow.com/questions/8660202/what-does-mean-for-java-generics